src/pymor/parameters/functionals.py

```diff
@@ -314,6 +330,42 @@ def d_mu(self, parameter, index=0):
         return self.with_(constant_value=0, name=self.name + '_d_mu')
 
 
+class LincombParameterFunctional(ParameterFunctional):
+    """A |ParameterFunctional| representing a linear combination of |ParameterFunctionals|.
+
+    The coefficients must be provided as scalars or as.
```

Comment:
There is some text missing here ...

Reply:
Ok, there is probably too much text. Have you thought about allowing `ParameterFunctionals` also as `coefficients`? Probably not a good idea?

Reply:
Yes too much text, thanks. I thought about it but since `ProductParameterFunctional` already do the job, I thought it might not be needed.